src/test/java/core/basesyntax/strategy/impl/SupplyTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.FruitsStorage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SupplyTest {
+    private static final FruitTransaction CORRECT_TRANSACTION
+            = new FruitTransaction(FruitTransaction.Operation.SUPPLY, "banana", 20);
+    private static final FruitTransaction CORRECT_FRUIT_THAT_NOT_EXIST
+            = new FruitTransaction(FruitTransaction.Operation.SUPPLY, "pineapple", 150);
```

Comment:
idk do you have any standards to evaluate only fruit that was in balance report, but i suggest to be more flexible)